apps/communications/contacts/js/views/details.js

```diff
@@ -82,6 +83,8 @@ contacts.Details = (function() {
 
     initPullEffect(cover);
 
+    LazyLoader.load('/contacts/js/nfc.js');
```

Comment:
+1. Move this to an upper level of abstraction (probably in Contacts when `initDetails` happen), and then we will move this directly to our self-contained view of `detail`